uint64 	id
geometry_msgs/Vector3  position
geometry_msgs/Vector3  goal
geometry_msgs/Vector3  velocity
